pressbooks/pressbooks-book
# Change color to increase contrast on white background

Fix for #1004 

To test:
1. Create a table with a caption
2. Inspect the caption to make sure that the caption color has a contrast of greater than 4.5:1 against its background
